From: Jiri Slaby <jslaby@suse.cz>
Subject: kABI: mm/swap.c, remove inclusion
Patch-mainline: never, kabi fix

3.4.69 commit b07ef016454ff46f98e633b5a6247ca7e343fb67 (mm: fix aio
performance regression for database caused by THP), upstream commit
7cb2ef56e6a8b7b368b2e883a0a47d02fed66911, added one #include into
mm/swap.c, but that made some symbols defined which kABI checker does
not like. So protect the include by GENKSYMS as usual.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 mm/swap.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/mm/swap.c
+++ b/mm/swap.c
@@ -30,7 +30,9 @@
 #include <linux/backing-dev.h>
 #include <linux/memcontrol.h>
 #include <linux/gfp.h>
+#ifndef __GENKSYMS__
 #include <linux/hugetlb.h>
+#endif
 
 #include "internal.h"
 
